elements/pl-big-o-input/pl-big-o-input.py

```diff
@@ -33,7 +33,7 @@ class BigOType(Enum):
 PLACEHOLDER_TEXT_THRESHOLD = 20
 SHOW_HELP_TEXT_DEFAULT = True
 WEIGHT_DEFAULT = 1
-DISPLAY_DEFAULT = bou.DisplayType.INLINE
+DISPLAY_DEFAULT = pl.DisplayType.INLINE
```

Comment:
I'm a little hesitant about making this globally shared. This is small enough (3 lines) where I really think this should just be duplicated for each element.

Reply:
I had second thoughts about this after I did it actually, will revert.